src/lib.rs

```
@@ -137,3 +144,44 @@ pub fn build_cosmogony(pbf_path: String) -> Result<Cosmogony, Error> {
     };
     Ok(cosmogony)
 }
+
+pub fn read_libpostal_yaml_folder(yaml_files_folder: String) -> BTreeMap<String, AdminRules> {
+    let mut admin_levels: BTreeMap<String, AdminRules> = BTreeMap::new();
+
+    if let Ok(paths) = fs::read_dir(yaml_files_folder) {
```

we don't want this to silently fail